tests/unit/models/gpflow/test_utils.py

```diff
@@ -37,50 +38,6 @@
 from trieste.models.optimizer import BatchOptimizer, Optimizer
 
 
-class _ModuleWithBijector(tf.Module):
-    def __init__(self) -> None:
-        super().__init__()
-        self.const = tf.constant(1.2)
-        self.var = tf.Variable(3.4)
-        self.bijector = tfp.bijectors.Exp()
-
-
-class _NestedModuleWithBijector(tf.Module):
-    def __init__(self) -> None:
-        super().__init__()
-        self.inst = _ModuleWithBijector()
-        self.bijector = tfp.bijectors.Sigmoid(0.0, 1.0)
-
-
-def test_module_deepcopy_is_copyable() -> None:
```

Comment:
we should keep a test that checks if the model can be deepcopied, right?

Reply:
This test is checking a sepcific issue with copying tf.Modules, not whether models are deepcopyable. The test we want to keep is `test_gaussian_process_deep_copyable`.

Reply:
ah, right, there are more deepcopy tests below